### PR TITLE
Marketplace Layout

### LINKED ISSUE #1683: Marketplace - App layout
## UI 
[Invision](https://projects.invisionapp.com/d/main?origin=v7#/console/19261929/402149797/preview?scrollOffset=0)
image

## Acceptance Criteria 
- [ ] Navbar
- [ ] Toolbar burger mode by default
Use same as dashboard

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
fix #1683

### CODE REVIEW

File: apps/catalog/catalog/src/app/marketplace/layout/layout.component.html [hunk 1]
```diff
@@ -1,3 +1,69 @@
+<mat-sidenav-container>
+  <mat-sidenav #sidenav>
+    <!-- SIDE NAVIGATION -->
+    <mat-nav-list>
+      <a mat-list-item routerLink="home" routerActiveLink="active">
```
Comment (jenna1k) [inline apps/catalog/catalog/src/app/marketplace/layout/layout.component.html]: I don't see `routerActiveLink="active"` works
we had the same issue last time and I remember bruce did some function about active link
if it should be refactored later, how about leaving TODO on it?

File: apps/catalog/catalog/src/app/marketplace/layout/layout.component.html [hunk 2]
```diff
@@ -1,3 +1,69 @@
...
+        <mat-icon mat-list-icon>home</mat-icon>
```
Comment (jenna1k) [inline apps/catalog/catalog/src/app/marketplace/layout/layout.component.html]: all these icons are comfirmed by mathilde? or should it be replaced later?

File: libs/notification/src/lib/notification/notification-widget/notification-widget.component.ts [hunk 3]
```diff
@@ -15,6 +15,7 @@ import { Invitation, InvitationStatus, InvitationType } from '@blockframes/invit
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class NotificationWidgetComponent implements OnInit {
+  @Input() page = 'search';
```
Comment (jenna1k) [inline libs/notification/src/lib/notification/notification-widget/notification-widget.component.ts]: I don't understand why it goes to search

File: libs/ui/src/lib/overlay-widget/overlay-widget.component.ts [hunk 4]
```diff
@@ -18,9 +18,61 @@ const fade = trigger('fade', [
...
+  encapsulation: ViewEncapsulation.None,
+  animations: [fade]
 })
 export class OverlayWidgetComponent {
   @ViewChild('ref', { static: false }) public ref: TemplateRef<any>;
 }
+
+
+// CARD
+@Directive({
```
Comment (jenna1k) [inline libs/ui/src/lib/overlay-widget/overlay-widget.component.ts]: nice! 👍

File: apps/catalog/catalog/src/app/marketplace/layout/layout.component.html (hunk 1 above)
Comment (GrandSchtroumpf) [reply apps/catalog/catalog/src/app/marketplace/layout/layout.component.html]: This is beause it's `routerLinkActive` not `routerActiveLink` ^^

File: apps/catalog/catalog/src/app/marketplace/layout/layout.component.html (hunk 2 above)
Comment (GrandSchtroumpf) [reply apps/catalog/catalog/src/app/marketplace/layout/layout.component.html]: I'll update with Mathilde's icons

File: libs/notification/src/lib/notification/notification-widget/notification-widget.component.ts (hunk 3 above)
Comment (GrandSchtroumpf) [reply libs/notification/src/lib/notification/notification-widget/notification-widget.component.ts]: Good point. Should be activity-feed. Thx

**Pull request merged**